src/CMakeLists.txt

```diff
@@ -3,6 +3,8 @@
 
 option(H264NAL_SMALL_FOOTPRINT, "xmall footprint build")
 
+add_definitions(-D__STDC_FORMAT_MACROS)
```

Comment:
I feel it's a better solution to explicit add "#define __STDC_FORMAT_MACROS" in any file where we have not defined it yet, rather that using a cmake flag. What do you think?